src/components/App/App.js

```diff
@@ -0,0 +1,28 @@
+require('normalize.css/normalize.css')
+require('./App.css')
+
+import React from 'react'
+import Navbar from '../Navbar/Navbar'
+import { Switch, Route } from 'react-router-dom'
+import Shop from '../../screens/Shop/Shop'
+import ShoppingCart from '../../screens/ShoppingCart/ShoppingCart'
+import ProductDetails from '../../screens/ProductDetails/ProductDetails'
+
+//let yeomanImage = require('../../images/yeoman.png');
+
+const App = () => {
+  return (
+    <div className="index">
+      <Navbar/>
+
+      <Switch>
```

Comment:
I'm using here also `<Router>` but in new version form: `<BrowserRouter>` (old `<Router history={browserHistory}>`) and in index.js file.

### Switch
Renders the first child `<Route>` or `<Redirect>` that matches the location.

### Router
The common low-level interface for all router components. Typically apps will use one of the high-level routers instead: `<BrowserRouter>`,[...].The most common use-case for using the low-level `<Router> `is to synchronize a custom history with a state management lib like Redux or Mobx.